package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "growthbook-proxy",
+  "name": "@growthbook/growthbook-proxy",
   "engines": {
     "node": ">=16"
   },
   "description": "GrowthBook proxy server for caching, realtime updates, telemetry, etc",
   "private": true,
-  "version": "1.0.2",
-  "main": "dist/index.js",
+  "version": "1.0.3",
+  "main": "dist/src/app.js",
```

Comment:
The compilation shouldn't be outputting a src directory inside of dist. Might have to see the rootDir option in tsconfig

Reply:
oh, that's left over from rollup, needed to create a subdir to keep separate from the esm/cjs folders. will fix...

Reply:
ah i see why it's still happening even after rollup removal...

We import package.json to automatically emit the version number. Because it's back a level from /src, it ends up preserving the /src folder. Is this a faux pas, as it seems to work?

Reply:
used require() and it seems to work fine. Hopefully won't mess up any esm compatibility